bedrock/mozorg/templates/mozorg/about/leadership/includes/senior.html

```diff
@@ -520,5 +520,80 @@ <h2 class="group-title">Senior Leadership</h2>
 
     </div>
   </article>
+
+  <article id="leyla-samiee" data-id="leyla-samiee" class="vcard has-bio" itemscope itemtype="http://schema.org/Person">
+    <figure class="headshot">
+      {{ resp_img(
+        url='img/mozorg/about/leadership/leyla-samiee.jpg',
+        srcset={
+          'img/mozorg/about/leadership/leyla-samiee-high-res.jpg': '2x'
+        },
+        optional_attributes={
+          'width': '160',
+          'height': '160',
+          'class': 'photo',
+          'itemprop': 'image'
+        }
+      ) }}
+      <figcaption><h4 class="fn" itemprop="name">Leyla Samiee</h4></figcaption>
+    </figure>
+
+    <div class="person-info">
+      <p class="title" itemprop="jobTitle">VP of Engineering</p>
+
+      <ul class="utility">
+        <li><a class="link" href="{{ url('mozorg.about.leadership.index') }}#leyla-samiee">Link to this bio</a></li>
+      </ul>
+    </div>
+
+    <div class="person-bio">
+      <p>
+        As Vice President of Engineering, Mozilla Social Leyla leads the
+        engineering team to help Mozilla reclaim the social internet. She is responsible
+        for managing a global team of Mozillians who are focusing on building a new
+        way of socializing on the web. Powered up with our centralized content platform
+        and AI engines, this major initiative will enable our customers to enjoy the best
+        social experiences guided by Mozilla's manifesto, values and key principles.
+      </p>
+
+      <p>
+        Before Mozilla, Leyla was a senior engineering leader at Meta, when she
+        and her organization owned Meta's ML/AI internal development platform
+        and tools. She built 0->1 number of key products serving ML development
```

Comment:
I'm just double-checking with the copywriter if `0->1` is to be correctly used here.